messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -620,6 +621,120 @@ class MessagingTest : BaseMessagingTest() {
         }
     }
 
+    @Test
+    fun testMyself() {
+        testInCoroutine {
+            newDB.use { dogDB ->
+                newMessaging(dogDB, "dog").with { dog ->
+                    val dogId = dog.myId.id
+                    dog.addOrUpdateDirectContact(dogId, "Note to self")
```

Comment:
it can work on the same test, but I believe it's better to have a separated case just for that scenario, if that's not necessary  then I can move it to the`testManageDirectContact`: https://github.com/getlantern/messaging-android/blob/090a369f2d10dd70a776e54e5f8ef49fb4d6ce6c/messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt#L49 @oxtoacart  👀

Reply:
I just mean that this kind doesn't really do anything because that contact already exists.  You can just remove it.